allauth/account/views.py

```diff
@@ -174,6 +176,39 @@ def get_context_data(self, **kwargs):
                     "redirect_field_value": redirect_field_value})
         return ret
 
+    def get_ajax_data(self):
+
+        enabled_providers = {}
+
+        if SOCIALACCOUNT_ENABLED:
+
+            for provider in providers.registry.get_list(self.request):
```

Comment:
In the current main branch, this should become `adapter.list_providers(request)`.